sources/packages/web/src/plugins/vuetify.ts

```diff
@@ -22,10 +22,10 @@ export default createVuetify({
           warning: "#ff7a00",
           success: "#16c92e",
           error: "#e4222e",
-          "info-bg": "#f0eef9",
-          "warning-bg": "#fff9ed",
-          "success-bg": "#eeffef",
-          "error-bg": "#fdf4f4",
+          "info-bg": "#a99fda",
+          "warning-bg": "#FFC38C",
```

Comment:
we will also need to change here, 
![image](https://user-images.githubusercontent.com/77353155/204057053-339603ef-bd71-4c94-adcf-353ec8d0c2a6.png)

Reply:
@ann-aot Thanks ann. The ones in Vuetify.ts change the badge border colours which are different from banner background colours. So I believe we should leave the $banner-background-info as is.

Reply:
vuetify theme is not only for badges border, it is for all the components with property `color` and `background color` like vuetify banners, buttons,snack bar, alerts etc. these styles variable are used wherever we can't use color or background-color property and sometimes even to override the vuetify default styles